dev/com.ibm.ws.jaxws.2.2.webcontainer_fat_extended/fat/src/com/ibm/ws/jaxws/fat/EnableSchemaValidationTest.java

```diff
@@ -0,0 +1,553 @@
+        LOG.info("testAddedElementGlobalSchemaValidationTrue - Response = " + response);
+
+        assertNull("Expected null response from server, but was" + response, response);
+        assertNotNull("Expected Unmarshalling Error: unexpected element exception is not thrown",
+                      server.waitForStringInLog("No child element is expected at this point."));
+    }
+
+    /**
+     * Test the global setting of enableSchemaValidation=true in server.xml, with the wrong element name where <response> element is expected
+     * which will cause an cvc-complex-type.2.4.a exception with the default schema validation enabled.
+     *
+     * Server config - global-true-server.xml
+     * Expected response - Unmarshalling Error: cvc-complex-type.2.4.a
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testWrongElementNameGlobalSchemaValidationTrue() throws Exception {
+        server.reconfigureServer("EnableSchemaValidationTestServer/global-true-server.xml", "CWWKG0017I");
+
+        // Visit the URL:  http://hostName:hostPort/testHandlerClient/IgnoreUnexpectedElementTestServiceServlet?target=user
+        String response = runTest(server, "testHandlerClient/IgnoreUnexpectedElementTestServiceServlet?target=WrongElementName");
+        // Log response to output.txt
+        LOG.info("testWrongElementNameGlobalSchemaValidationTrue - Response = " + response);
+
+        assertNull("Expected null response from server, but was" + response, response);
+        assertNotNull("Expected Unmarshalling Error: cvc-complex-type.2.4.a  in server logs",
+                      server.waitForStringInLog("cvc-complex-type.2.4.a"));
+    }
+
+    /**
+     * Test the global setting of enableSchemaValidation=true in server.xml, with the wrong namespace in the <response> element
+     * which will cause an cvc-complex-type.2.4.a exception with the default schema validation enabled.
+     *
+     * Server config - global-true-server.xml
+     * Expected response - Unmarshalling Error: cvc-complex-type.2.4.a
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testWrongNamespaceGlobalSchemaValidationTrue() throws Exception {
+        server.reconfigureServer("EnableSchemaValidationTestServer/global-true-server.xml", "CWWKG0017I");
+
+        // Visit the URL:  http://hostName:hostPort/testHandlerClient/IgnoreUnexpectedElementTestServiceServlet?target=user
+        String response = runTest(server, "testHandlerClient/IgnoreUnexpectedElementTestServiceServlet?target=WrongNamespace");
+        // Log response to output.txt
+        LOG.info("testWrongNamespaceGlobalSchemaValidationTrue - Response = " + response);
+
+        assertNull("Expected null response from server, but was" + response, response);
+        assertNotNull("Expected Unmarshalling Error: cvc-complex-type.2.4.a  in server logs",
+                      server.waitForStringInLog("cvc-complex-type.2.4.a"));
+    }
+
+    /**
```

Comment:
Isn't this same as test #1 - additional element in response?

Reply:
While the exception is the same, the point is to make sure that the unmarshaller will pick up that the namespace + element name makes in an unexpected element